### ISSUE TITLE
Please fix warnings while compiling in Elixir 1.8

### ISSUE THREAD

**Issue by Maples7** [rentpath/ex_json_logger#15]
```
warning: System.stacktrace/0 outside of rescue/catch clauses is deprecated. If you want to support only Elixir v1.7+, you must access __STACKTRACE__ inside a rescue/catch. If you want to support earlier Elixir versions, move System.stacktrace/0 inside a rescue/catch
  lib/plug/conn/wrapper_error.ex:23
warning: function :telemetry.execute/3 is undefined (module :telemetry is not available)
Found at 2 locations:
  lib/plug/telemetry.ex:70
  lib/plug/telemetry.ex:74
```
See https://travis-ci.org/seniverse/ex_json_logger_examples/builds/538186741 for details.

**Comment by Stratus3D** [comment]
I don't see any compiler warnings from the code in ex_json_logger itself. I see some warnings from some of the dependencies. We should probably update the dependencies. Specifically plug and poison.

**Comment by Maples7** [comment]
I see. Look forward to it! 👐  Maybe fix it with #13? 
I can do some help if you need. We 'd like to use this package in our system.

**Comment by Stratus3D** [comment]
We should be tagging version 0.2.0 sometime in the next coming weeks. A PR to upgrade the dependencies has been merged, so all the warnings we can address have been addressed.
